src/Arcus.WebApi.Logging/RequestTrackingMiddleware.cs

```diff
@@ -210,7 +211,26 @@ private void LogRequest(string requestBody, string responseBody, HttpContext htt
             Dictionary<string, object> telemetryContext = CreateTelemetryContext(requestBody, responseBody, httpContext.Request.Headers, _logger);
             telemetryContext.Add("Body", "Request body is now available in 'RequestBody' dimension");
 
-            _logger.LogRequest(httpContext.Request, httpContext.Response, duration, telemetryContext);
+            var operationName = BuildOperationName(httpContext);
```

Comment:
If we let the `operationName` be `null` we might consider renaming this from `Build...` to `Determine...`, `Get...` as we are not building anything anymore.

Reply:
Changed this.